spec/react_on_rails/utils_spec.rb

```
@@ -100,88 +208,128 @@ module ReactOnRails
-              .and_return(false)
-            allow(ReactOnRails::PackerUtils).to receive_message_chain("packer.manifest.lookup!")
-              .with("server-bundle.js")
-              .and_return("webpack/development/server-bundle-123456.js")
-            allow(ReactOnRails::PackerUtils).to receive_message_chain("packer.dev_server.running?")
-              .and_return(true)
-
-            path = described_class.server_bundle_js_file_path
-
-            expect(path).to end_with("/public/webpack/development/server-bundle-123456.js")
+        describe ".rsc_bundle_js_file_path with #{packer_type} enabled" do
+          let(:packer_public_output_path) { Pathname.new("public/webpack/development") }
```

rubocop (1.69.1)

[convention] Add an empty line after the last `let`.

(RSpec/EmptyLineAfterFinalLet)